### ISSUE TITLE
关于DIEN的一个bug

### ISSUE THREAD

**Issue by sudongxiang** [alibaba/x-deeplearning#170]
在DIEN的文件script/utils.py 中，返回值scores的shape为（B，1，T），而实际上，应为（B，T），否则，代码无法正常运行。  只需要加上一行scores  =   tf.reduce_sum(scores,axis=1) 即可。
